samples/snippets/cloud-client/src/main/java/recaptcha/passwordleak/CreatePasswordLeakAssessment.java

```diff
@@ -0,0 +1,214 @@
+
+// [START recaptcha_enterprise_password_leak_verification]
+
+import com.google.cloud.recaptcha.passwordcheck.PasswordCheckResult;
+import com.google.cloud.recaptcha.passwordcheck.PasswordCheckVerification;
+import com.google.cloud.recaptcha.passwordcheck.PasswordCheckVerifier;
+import com.google.cloud.recaptchaenterprise.v1.RecaptchaEnterpriseServiceClient;
+import com.google.protobuf.ByteString;
+import com.google.recaptchaenterprise.v1.Assessment;
+import com.google.recaptchaenterprise.v1.CreateAssessmentRequest;
+import com.google.recaptchaenterprise.v1.Event;
+import com.google.recaptchaenterprise.v1.PrivatePasswordLeakVerification;
+import com.google.recaptchaenterprise.v1.TokenProperties;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.stream.Collectors;
+
+public class CreatePasswordLeakAssessment {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException {
+    // TODO(developer): Replace these variables before running the sample.
+    // GCloud Project ID.
+    String projectID = "project-id";
+
+    // Site key obtained by registering a domain/app to use recaptcha Enterprise.
+    String recaptchaSiteKey = "recaptcha-site-key";
+
+    // The token obtained from the client on passing the recaptchaSiteKey.
+    // To get the token, integrate the recaptchaSiteKey with frontend. See,
+    // https://cloud.google.com/recaptcha-enterprise/docs/instrument-web-pages#frontend_integration_score
+    String token = "recaptcha-token";
+
+    // Action name corresponding to the token.
+    String recaptchaAction = "recaptcha-action";
+
+    checkPasswordLeak(projectID, recaptchaSiteKey, token, recaptchaAction);
+  }
+
+  /*
+   * Detect password leaks and breached credentials to prevent account takeovers (ATOs)
+   * and credential stuffing attacks.
+   * For more information, see: https://cloud.google.com/recaptcha-enterprise/docs/getting-started
+   * and https://security.googleblog.com/2019/02/protect-your-accounts-from-data.html
+   * Steps:
+   * 1. Use the 'createVerification' method to hash and Encrypt the hashed username and password.
+   * 2. Send the hash prefix (2-byte) and the encrypted credentials to create the assessment.
+   * (Hash prefix is used to partition the database.)
+   * 3. Password leak assessment returns a database whose prefix matches the sent hash prefix.
+   * Create Assessment also sends back re-encrypted credentials.
+   * 4. The re-encrypted credential is then locally verified to see if there is a
+   * match in the database.
+   *
+   * To perform hashing, encryption and verification (steps 1, 2 and 4),
+   * reCAPTCHA Enterprise provides a helper library in Java.
+   * See, https://github.com/GoogleCloudPlatform/java-recaptcha-password-check-helpers
+   *
+   * If you want to extend this behavior to your own implementation/ languages,
+   * make sure to perform the following steps:
```

Comment:
Adding a single line w/ a * would make this easier to read.

Reply:
Done. Thanks.